src/omnistap/OmnisTAP/ogTAPSuper/$mock.omh

```diff
@@ -65,6 +65,7 @@ If isnull(lirMockClass)
 	Do lirConstruct.$lvardefs.$add("porMock",kObjectref,#NULL,0,kTrue)
 	Do lirConstruct.$methodlines.$add("Calculate _iorMock as porMock")
 	Do lirConstruct.$methodlines.$add("Do _iorMock.$setMockInstance($cinst())")
+	Do lirConstruct.$methodlines.$add("Do inherited")

```

Comment:
This looks like additional new behavior. Would you please add this in a separate pull request with a new parameter to `$mock` so the developer can control if they want inherited code to run?